entries/a/appfigures.com.json

```
@@ -0,0 +1,12 @@
+{
+  "AppFigures": {
+    "domain": "appfigures.com",
+    "contact": {
+      "twitter": "appfigures",
+      "form": "https://appfigures.com/support/contact"
+    },
+    "categories": [
+      "developer"
```

I suppose "Marketing & Analytics" fits better here.
```suggestion
      "marketing"
```